website/content/en/docs/User Guide/Cost Explained/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Breaking Down Cloud Operations Sandbox Cost"
+linkTitle: "Sandbox Cost Explained"
+weight: 2
+---
+{{% pageinfo %}}
+* [Cloud Operations Sandbox Cost Estimations](#Cloud-Operations-Sandbox-Cost-Estimations)
+  * [GCP Calculator](#GCP-Calculator)
+  * [Sandbox Cost Estimations](#Sandbox-Cost-Estimations)
+  * [Sandbox Compute Cost Estimations](#Sandbox-Compute-Cost-Estimations)
+  
+ 
+## Cloud Operations Sandbox Cost Estimations
+Cloud Operations Sandbox Cost is combined from compute cost, storage and usage of network and additional services like Cloud Operations Suite. While some of the cost, like GKE infrastructure is constant, for some services cost is a factor of usage. Note, that some of Sandbox costs fall under the [free tier](https://cloud.google.com/free/docs/gcp-free-tier#free-tier), but not not all.
+To learn more about billing and pricing please refer to: [Google Cloud Pricing Overview](https://cloud.google.com/pricing/), [Google’s Pricing philosophy](https://cloud.google.com/pricing/philosophy/), [GCP's Billing](https://cloud.google.com/billing/docs).
+
+To view billing information for your project in the console go to [billing.](https://console.cloud.google.com/billing)
+![image](/docs/images/user-guide/57-billing-menu.png)
+
+![image](/docs/images/user-guide/58-billing-console.png)
+ 
+### GCP Calculator
+GCP has transparent pricing which is applied per resource. You can plan and estimate pricing using the [pricing calculator](https://cloud.google.com/products/calculator). Some of the really cool features of Google cloud platform are sustained use discounts that are applied as credits to your account rather than just lower costs. That allows us later to see exact costs attributed to usage, so we can more accurately reduce usage later.
+
+### Sandbox Cost Estimations
+To break down the cost we will use a GCP calculator. As network, storage and [observability cost](https://cloud.google.com/stackdriver/pricing) are very minimal due to the low volume, in the next section we will focus on the cost of the infrastructure. 
+
+#### Sandbox Compute Cost Estimations
+You can view the provisioned resources in [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform) you can see the exact resources created or in GCP's console.
+
+![image](/docs/images/user-guide/59-resources-console.png)
+
+In order to use the calculator you should first select a product from the scrolling list and fill the needed variables. We should do the same for each one of the products.
+ 
+**Resources Information**
```

Comment:
Personally, I don't think we need all this information on how to fill out the calculator. I think we can just show [the image of the resources used](/docs/images/user-guide/59-resources-console.png), then show the [images of the calculator all filled in](/docs/images/user-guide/64-cost-est.png), then explain which variables might be most likely to grow with usage. (And it would be great if we could include networking costs)

Reply:
Is it possible to link to a filled-in calculator for users to experiment with?

Reply:
AFAIk no, you can send in email, share with specific people (email) or save with your account.
I saw that when you save to your account it saves an id and I could access incognito and from private account, but I dont know for how long it is saved https://cloud.google.com/products/calculator#id=ac05b51f-d0b0-43ae-a594-03f99feb8bed